### DISCUSSION TITLE
Network Owner Checker

### DISCUSSION THREAD

**Discussion by Canglangzi** [FirstGearGames/FishNet#711] [Feature Request]
I want a component that can force this netobj to become the owner. I can use this functionality when I want to generate a network object with client permissions and only show it to the owner such as an interactive UI or NPC. Since clients can invoke commands on objects they have access to, such objects can have network behaviors that include commands invoked directly by external clients.

**Comment by FirstGearGames** [comment]
I don't fully understand your request. If you want an object only visible to owner you can use the OwnerOnly observer condition.
https://fish-networking.gitbook.io/docs/manual/components/network-observer#conditions

**Reply by Canglangzi** [reply]
I understand thank you
